source/_integrations/livisi.markdown

```diff
@@ -40,3 +41,7 @@ The current integration will not find your SHC automatically and needs to be con
 ## Device Discovery
 
 All devices are automatically discovered and included by the integration. If you include a new device in LIVISI SmartHome, the device will automatically appear in Home Assistant after a few minutes.
+
+## Remotes and wall buttons
+
+Livisi remotes such as the 8 Button remote (BRC8) and wall buttons (WSC2) are stateless devices, meaning that they do not have a on/off state like regular entities in Home Assistant. Instead, such devices emit the event `livisi_event` when a button is pressed. You can test what events come in using the event {% my developer_events title="developer tools in Home Assistant" %} and subscribe to the `livisi_event`. Once you know what the event data looks like, you can use this to create automations.
```

Comment:
We should implement device triggers instead.

<https://developers.home-assistant.io/docs/device_automation_trigger>

Reply:
I read about device triggers and then wanted to look for an example in one of the other integrations. That's when I found this snippet in the Hue integration docs: https://www.home-assistant.io/integrations/hue/#hue-remotes-and-switches. As Hue is one of the very mature integrations, I figured that device triggers cannot be used when there are no entities. 
Do you happen to know an example where device triggers are used to model stateless hardware buttons?

Reply:
> As Hue is one of the very mature integrations

Hue is mostly old and has history, not all integration implement all modern things yet.

> I figured that device triggers cannot be used when there are no entities.

Nope, device triggers are not limited to entities at all.